src/containers/ImportResources/ImportResources.js

```diff
@@ -100,9 +119,18 @@ export class ImportResources extends Component {
     const gitresourcename = 'git-source';
     const gitcommit = 'master';
 
-    if (repourl === '' || !namespace) {
+    const valid = validateURL(repourl);
+    if (valid === false || !namespace) {
+      if (valid === false && repourl === '') {
+        this.setState({
+          invalidInput: repourl === ''
```

Comment:
Wait what?

Reply:
Theres a wierd thing for the dashboard that if the repourl is empty it wont show up the error unless its specifically done setting the repourl to '' 
If i take it out then if the repourl is empty then when you press submit it wont let you submit but wont show up an error as the repourl hasnt got anything there 
The if statement is only needed for when the repourl is empty because the errors show up fine for everything else in the page but not when it is empty or contains only a space

Reply:
Alrighty, I'd use a more descriptive variable name than `valid` then and see my below comment about that repo url being empty bit (evaluating to true which I assume invalidInput wants) - again I'd prefer a few more lines to improve readability over conciseness, favour tidyness and readability where you can please